grails-app/services/au/org/emii/netcdfsubset/SubsetService.groovy

```diff
@@ -12,17 +12,30 @@ class SubsetService {
         def generator = _getGenerator()
 
         try {
+            _writeTemplateToLayerConfigDir(typeName)
             generator.write(typeName, cqlFilter, dataSource.connection, response)
         }
         catch (Exception e) {
-            log.error "An error has occurred when writing netcdf file", e
+            log.error "An error has occurred when generating netcdf file", e
         }
     }
 
     def _getGenerator = { ->
          new NcdfGenerator(
-                grailsApplication.config.netcdf_filters.layer_config_dir,
-                System.getProperty('java.io.tmpdir')
+             grailsApplication.config.ncdfgenerator.layerConfigDir,
+             System.getProperty('java.io.tmpdir')
         )
     }
+
+    def _writeTemplateToLayerConfigDir(typeName) {
```

Comment:
Seems an odd place to do this.  We'll be writing out the config file for every request